src/apiCall.ts

```diff
@@ -16,18 +27,11 @@ export const filterItems = async (url: string) => {
   return response.json()
 };
 
-export const createItem = async (data: any) => {
-  // we cannot leave data: any!! We should make an interface 
+export const createItem = async (data: FormData) => {
   const url = `https://closet-manager-be.herokuapp.com/api/v1/users/1/items`;
-    // hard-coding this fetch for user 1 for now, make dynamic if we add other users
   const response = await fetch(url, {
     method: "POST",
-    // "There is no "Content-Type" key in the headers — the 
-    // content type is multipart/form-data,
-    //  which is implied by the FormData object itself"
     body: data,
-    // "The body is not stringified. The FormData API handles all 
-    // the necessary processing for the image to be sent over the web."
   });
   if (!response.ok) {
     console.log(response)
```

Comment:
If response is not ok, maybe we use a `console.error()` or `console.warn()` instead of log?